hr_timesheet_clean/wizards/timesheet_clean.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="timesheet_clean_form_view" model="ir.ui.view">
+        <field name="name">timesheet.clean.form (in hr_timesheet_clean)</field>
+        <field name="model">timesheet.clean</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <group>
+                        <field name="date_from" />
+                        <field name="date_to" />
+                    </group>
+                    <group />
```

Comment:
I'm not sure this line has an impact in the view. Could you confirm it? 
I can't test on the runboat since it  was not trigger by the PR :/